doc/cephfs/client-auth.rst

```diff
@@ -153,3 +153,41 @@ The optional ``{network/prefix}`` is a standard network name and
 prefix length in CIDR notation (e.g., ``10.3.0.0/16``).  If present,
 the use of this capability is restricted to clients connecting from
 this network.
+
+File system Information Restriction
+===================================
+
+If desired, the monitor cluster can present a limited view of the file systems
+available. In this case, the monitor cluster will only inform clients about
+filesystems specified by the administrator. Other filesystems will not be
+reported and commands affecting them will be ignored as if the filesystems
+did not exist.::
+
+ client.0
+     key: AQAz7EVWygILFRAAdIcuJ12opU/JKyfFmxhuaw==
+     caps: [mon] allow rw fsid=1, allow rw fsid=2
```

Comment:
Names are more memorable than IDs in general, so using names would be better IMO.

Would it be OK to allow both, FS names and IDs? It would be more convenient, I guess.

Reply:
Both would also work fine (try to parse fscid first) but might needlessly complicate the code. 

Note: using the fs name is consistent with other caps like OSD caps which use pool names. If a pool is deleted and recreated, the old caps still work because they work off the name. So, I would lean towards just using the fsname.

Reply:
>  If a pool is deleted and recreated, the old caps still work because they work off the name. So, I would lean towards just using the fsname.

Okay. I'll modify the code to support FS names only.

Reply:
Incorporated the change.